bin/snapshot-backups/site-snapshot-download

```diff
@@ -12,13 +12,24 @@ fi
 if [ -f ./env.local ]; then
   . ./env.local
 fi
+if [ -f $SCRIPT_DIR/../deploy/includes ]; then
+  . $SCRIPT_DIR/../deploy/includes
+else
+   echo Cannot find $SCRIPT_DIR/../deploy/includes
+   exit 1
+fi
 
 # just testing?
-if [ x$1 == x"--dryrun" ]; then
-  export echo=echo
+if [ x$1 = x"--dryrun" ]; then
+  echo=echo
   shift
 fi
 
+SPACE=${1:-please-provide-space-name-as-first-argument}
+SPACE=${SPACE,,}
+assertCurSpace $SPACE
+shift
+
```

Comment:
Usage statement coming up on line 36 needs to be updated to reflect the new reality. (GitHub, let me comment on lines that weren't changed!)

Reply:
Done.  thx!